docs/sources/setup-grafana/configure-security/configure-authentication/jwt/index.md

```diff
@@ -36,9 +36,7 @@ header_name = X-JWT-Assertion
 
 ## Configure login claim
 
-To identify the user, some of the claims needs to be selected as a login info. You could specify a claim that contains either a username or an email of the Grafana user.
-
-Typically, the subject claim called `"sub"` would be used as a login but it might also be set to some application specific claim.
+To identify the user, some of the claims needs to be selected as a login info. The subject claim called `"sub"` is mandatory and needs to specify a claim that contains either a username or an email of the Grafana user.
```

Comment:
The `sub` claim does not necessarily contains the username or email: "The subject value MUST either be scoped to be locally unique in the context of the issuer or be globally unique". https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.2
```suggestion
To identify the user, some of the claims needs to be selected as a login info. The subject claim called `"sub"` is mandatory and needs to identify the principal that is the subject of the JWT. 

Typically, the subject claim called `"sub"` would be used as a login but it might also be set to some application specific claim.
```

Reply:
I have signed the CLA.